styles/alternativeto/catppuccin.user.css

```diff
@@ -0,0 +1,506 @@
+      );
+      content: url("data:image/svg+xml,@{svg}");
+    }
+    // hanburger menu
+    [class^="DropdownListLink_additionalInfo"] {
+      color: @subtext0;
+    }
+    // svg color variables
+    @dark-accent-color: desaturate(darken(@accent-color, 25%), 50%);
```

Comment:
This color is pretty insane lol. What does it look like if we just use accent? Same with the others below - if we modify them this much they are no longer Catppuccin.

Reply:
Just this color here before we can merge. Is this for the SVG on the home page? How does it look with just normal accent?

Reply:
![h](https://github.com/user-attachments/assets/a018ca4d-cfd5-4949-aba8-6b338d3f58bb)

Reply:
Can we just do `@accent-color`? Thanks.